scss/_settings_colors.scss

```diff
@@ -403,7 +403,7 @@ $colors--theme--button-negative-text: var(--vf-color-button-negative-text);
 
   // SCSS variables need to be interpolated to work in CSS custom properties
   --vf-color-background-default: #{$color-paper};
-  --vf-color-background-alt: #{$color-x-light};
+  --vf-color-background-alt: #{#ebebeb};
```

Comment:
Should we define a variable for it? :thinking:

Reply:
Good question, there don't seem to be a need yet. We are also planning to work on design tokens soon, which will end up as better abstracted colour definitions.

Reply:
:+1: LGTM then